docs/contributor-guide/developer-guide/technology-overview.md

```diff
@@ -16,32 +16,6 @@ All view classes should extend [`BaseHtmlView`](https://github.com/civiform/civi
 
 The `View` classes are generally organized by which role(s) they are viewable by (e.g., [app/view/admin/](https://github.com/civiform/civiform/tree/main/server/app/views/admin) for pages viewable by Admins, [app/views/applicant/](https://github.com/civiform/civiform/tree/main/server/app/views/applicant) for pages viewable by Applicants). Each of these roles also has its own [`Layout` class](https://github.com/civiform/civiform/blob/main/server/app/views/admin/AdminLayout.java) that extends `BaseHtmlLayout` for rendering page content in the context of that role.
 
-## How to add WebJar dependencies
```

Comment:
Why delete this content completely?

Reply:
We don't use WebJars. Looks like we used for azure JS but later switched to loading JS from CDN. Going forward we are going to import JS libraries through webpack/npm and not WebJars.